app/concerns/mdm/workspace/boundary_range.rb

```diff
@@ -38,12 +38,31 @@ def allow_actions_on?(ips)
 
     # Validates that {#boundary} is {#valid_ip_or_range? a valid IP address or
     # IP address range}. Due to this not being tested before it was moved here
-    # from Mdm, the default workspace does not validate. We therefore don't
-    # validate boundaries of workspaces that don't use them.
+    # from Mdm, the default workspace does not validate. We always validate boundaries
+    # and a workspace may have a blank default boundary.
     #
     # @return [void]
     def boundary_must_be_ip_range
-      errors.add(:boundary, "must be a valid IP range") unless !limit_to_network || valid_ip_or_range?(boundary)
```

Comment:
We always want to run the boundary check, as per the original behavior. Limit to network is only ever used to validate boundary constraints in the allow_actions_on? method which is used in a module to verify a user provided range meets the workspace defined boundaries. (limit_to_network = true)